content/docs/Other guides/customizing-rqd.md

```diff
@@ -11,7 +11,7 @@ This guide describes how to customize the default [RQD container image published
 on Docker Hub](https://hub.docker.com/r/opencue/rqd). The default RQD container
 image doesn't include any rendering software. This guide explains how to create
 a custom Dockerfile that builds on the basic `opencue/rqd` image to install
-rendering software. The sample code illustrates the steps using version 2.79 of
+rendering software. The sample code illustrates the steps using version 3.3 of
```

Comment:
I would drop the version number from this sentence, it'll become outdated again if/when we upgrade in the future.

Reply:
Noted. 
Removed in https://github.com/AcademySoftwareFoundation/opencue.io/pull/276/commits/7281a972a3d2a26ef67ef18e7cb3391e9627ae72